Use: integration type: rephrase to make more procedural (#503)
hacs/documentation: source/docs/use/dashboard.md

Each line in the data table, represent a repository that is (or can be) managed by HACS.

By default, it only shows you which repositories you have downloaded with HACS.

If you want to see more, use the [search](#search) and [filter](#filter) capabilities.

## Downloading a repository

1. In Home Assistant, open HACS and find the repository of interest. 

>     - If you want to browse by type, [filter](/docs/use/dashboard.md#filter), for example for type *integration*.

Author: ```suggestion
    - If you are unsure what you are looking for and want to browse, you can [filter](/docs/use/dashboard.md#filter). For example for type *integration*.

```

Each line in the data table, represent a repository that is (or can be) managed by HACS.

By default, it only shows the repositories you have downloaded with HACS.

If you want to see more, use the [search](#search) and [filter](#filter) capabilities.

## Downloading a repository

1. In Home Assistant, open HACS and find the repository of interest.
    - If you are unsure what you are looking for and want to browse, you can [filter](/docs/use/dashboard.md#filter). For example, for type *integration*.

> 

Reviewer: ```suggestion
```

    - If you already know what you want, type the name into the search field.
2. Select the repository from the list and read its documentation.
3. To download the repository, select **Download**.

>     - The repository is now added to your [Home Assistant configuration directory](https://www.home-assistant.io/docs/configuration.md#to-find-the-configuration-directory).

Reviewer: ```suggestion
    - The files provided by the repository is now added to your [Home Assistant configuration directory](https://www.home-assistant.io/docs/configuration.md#to-find-the-configuration-directory).
```
You are not getting the entire repo, only the important parts.

## Filter

> If you are just looking to browse repositories, you can do so with filtering.

Reviewer: ![image](https://github.com/hacs/documentation/assets/15093472/ff0c25de-d9f3-43d1-8d4f-1340006d5db5)
